NEWS

```diff
@@ -53,9 +53,29 @@ included in the vX.Y.Z section and be denoted as:
    (** also appeared: A.B.C)  -- indicating that this item was previously
                                  included in release version vA.B.C.
 
-Master (not on release branches yet)
-------------------------------------
-
+3.1.0 -- January, 2017
```

Comment:
Man, I read this too quickly this morning: shouldn't that be 2018?

Reply:
I figured inventing time travel was easier than getting an OMPI release out.  Didn't I tell you that's why I've been so behind in pull requests? :)

Yeah, fixed.